msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -357,6 +365,15 @@ class NativeAuthPublicClientApplication(
                             channel = result.challengeChannel
                         )
                     }
+                    is INativeAuthCommandResult.InvalidUsername -> {
```

Comment:
See comment in https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2297/files#r1466685798 - in the commits in common right now, this result is never returned.

Reply:
Updated